### DISCUSSION TITLE
More 2FA options

### DISCUSSION THREAD

**Discussion by Bulmer** [webmin/webmin#2050] [Ideas]
If someone could advise about how to go about creating another 2FA option for webmin, namely 2FAS, I could make a pull request and try? It's a very well thought out alternative to Authy and Google Authenticator.

**Comment by jcameron** [comment]
Adding a new authenticator would be great! Adding a new one isn't too hard actually, you just need to edit https://github.com/webmin/webmin/blob/master/webmin/twofactor-funcs-lib.pl and add functions similar to the current ones that end with _authy or _totp

**Comment by Bulmer** [comment]
Ok, I'll see how I go and let you know. If successful I'll do a pull
request.
